extensions/amp-lightbox/1.0/component.jss.js

```diff
@@ -38,39 +38,61 @@ before allowing buttons to be activated. */
 };
 
 const wrapper = {
-  top: 0,
-  bottom: 0,
-  left: 0,
-  right: 0,
-  width: '100%',
-  height: '100%',
-  position: 'fixed',
-  boxSizing: 'border-box',
-};
+  top: '0 !important',
+  left: '0 !important',
+  bottom: '0 !important',
+  right: '0 !important',
+  width: '100% !important',
```

Comment:
IMHO `width` and `height` shouldn't be `!important`. It's very reasonable go give a lightbox a smaller size. What could be useful is `margin:auto` to keep it centered by default.

FWIW, I'm not sure `!important` is particularly critical for all other props either. Especially since these styles are behind shadow.

Reply:
I'll remove for now. For some reason I thought we wanted them this way since they were in sidebar, but I understand how that can be different as we don't actually expect its backdrop to change.